apps/web/src/pages/TalentNominations/NominateTalent/components/Details.tsx

```diff
@@ -1,43 +1,655 @@
+                      })}
+                    />
+                    <Input
+                      type="email"
+                      id="advancementReferenceFallbackWorkEmail"
+                      name="advancementReferenceFallbackWorkEmail"
+                      rules={{
+                        required: intl.formatMessage(errorMessages.required),
+                      }}
+                      label={intl.formatMessage({
+                        defaultMessage: "Reference's work email",
+                        id: "5fMhLS",
+                        description:
+                          "Label for the text input for the reference's work email",
+                      })}
+                    />
+                  </div>
+                  <ClassificationInput
+                    name="advancementReferenceFallbackClassification"
+                    classificationsQuery={unpackMaybes(
+                      options?.classifications,
+                    )}
+                    rules={{
+                      group: {
+                        required: intl.formatMessage(errorMessages.required),
+                      },
+                      level: {
+                        required: intl.formatMessage(errorMessages.required),
+                      },
+                    }}
+                  />
+                  <Select
+                    id="advancementReferenceFallbackDepartment"
+                    name="advancementReferenceFallbackDepartment"
+                    rules={{
+                      required: intl.formatMessage(errorMessages.required),
+                    }}
+                    label={intl.formatMessage({
+                      defaultMessage: "Reference's department or agency",
+                      id: "lgF8zK",
+                      description: "Label for a reference's department",
+                    })}
+                    nullSelection={intl.formatMessage(
+                      uiMessages.nullSelectionOption,
+                    )}
+                    options={unpackMaybes(options?.departments).map(
+                      (department) => ({
+                        value: department.id,
+                        label: department.name.localized ?? "",
+                      }),
+                    )}
+                  />
+                </>
+              )}
+            </div>
+          )}
+          {lateralMovement && (
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+            >
+              <div>
+                <p data-h2-font-weight="base(700)">
+                  {intl.formatMessage({
+                    defaultMessage: "Lateral movement options",
+                    id: "4nhr77",
+                    description:
+                      "Title for lateral movement options section in nominations details step",
+                  })}
+                </p>
```

Comment:
Shoudn't this be a heading? It looks like one to me

Reply:
Off the figma design it's just regular body text, but ya it does look like one
![image](https://github.com/user-attachments/assets/831c8167-b028-40d6-a66f-56f903bc7c24)

Reply:
I agree that it should probably at least be a semantic heading.